spec/features/user/signup_spec.rb

```diff
@@ -0,0 +1,50 @@
+require 'rails_helper'
+
+RSpec.describe "user" do
+  it "can sign up with email" do
+    visit root_path
+    click_link_or_button('Sign up')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.